From 45d174ad5eb8102896058022c908f0cccec2837d Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Tue, 13 Apr 2021 20:36:10 -0400
Subject: [PATCH 11/36] Set ro.lmk.kill_timeout_ms

This prevents lmkd to be over-zealous and rapidly kill many processes,
because the process of recovering memory may need some time.

See https://cs.android.com/android/_/android/platform/system/memory/lmkd/+/30854e70d994c33130247126665f996f6f6fd6b4
---
 device.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/device.mk b/device.mk
index 665f17e..9ad7f0d 100644
--- a/device.mk
+++ b/device.mk
@@ -26,7 +26,8 @@ PRODUCT_PROPERTY_OVERRIDES := \
     media.sf.extractor-plugin=libffmpeg_extractor.so \
     debug.stagefright.ccodec=0 \
     debug.stagefright.omx_default_rank.sw-audio=1 \
-    debug.stagefright.omx_default_rank=0
+    debug.stagefright.omx_default_rank=0 \
+    ro.lmk.kill_timeout_ms=100
 
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES := \
     ro.arch=x86 \
-- 
2.17.1

